corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -176,6 +178,45 @@ def delete_index(self, cname):
 
         es_manager.index_delete(older_index)
 
+    def estimate_disk_space_for_reindex(self, stdout=None):
+        indices_info = es_manager.indices_info()
+        index_cname_map = self._get_index_name_cname_map()
+        index_size_rows = []
+        total_size = 0
+        for index_name in index_cname_map.keys():
+            meta = indices_info[index_name]
+            index_size_rows.append([
+                index_cname_map[index_name],
+                index_name,
+                self._format_bytes(int(meta['size_on_disk'])),
+                meta['doc_count']
+            ])
+            total_size += int(meta['size_on_disk'])
+        self._print_table(index_size_rows, output=stdout)
+        recommended_disk = total_size + (total_size * .2)  # 20% more that what is used
```

Comment:
Am I understanding correctly that `total_size` is the total space used by all existing indices? Would this recommendation be correct if only a few of them are being reindexed? Possibly even just the smallest one?

Also, nit
```suggestion
        recommended_disk = total_size * 1.2  # 20% more that what is used
```